server/src/challenge/challenge.e2e-spec.ts

```diff
@@ -128,6 +130,10 @@ describe('ChallengeModule E2E', () => {
     });
   });
 
+  /** describe groups your tests together you can nest groups together
```

Comment:
This is easily available with jest documentation, why specifically for the challenge module?